### DISCUSSION TITLE
[Help] - How can I properly run a Textual app on Windows ?

### DISCUSSION THREAD

**Discussion by CryptoNawwa** [Textualize/textual#779] [Q&A] [answered]
Hello everyone,

I built a textual app (source code here : [Github link](https://github.com/CryptoNawwa/bybit_scalper_tool)), it runs very smoothly on MacOs terminal but I can't make it work on windows

Problems I have : 
- When I write in the text input it just overlap with the header and messes with the display
- All the widgets are just messed up and merging with each other
- If I resize the termina once it show the proper UI but when I resize again the display is totally messed up and can't return to "normal" 

I tried on Powershell, CMD, Powershell7, Gitbash..

Can't try on Windows Terminal because I don't have the microsoft store but I don't think It will change a lot

Any idea ? What can I do to make it work better ? It seems like everyone is happy with the Windows support but in my case nothing is working

**Comment by sergi0g** [comment, marked as answer]
I think Textual was made for working on all platforms, but the default windows terminal does not support many colors and cannot render some characters properly, as seen in many issues. If your preferred platform is Windows, you should definitely use the Windows Terminal app from the Microsoft Store.
